src/registrar.js

```diff
@@ -74,9 +74,11 @@ class Registrar {
    * @returns {void}
    */
   onConnect (peerInfo, conn) {
-    if (!PeerInfo.isPeerInfo(peerInfo)) {
-      throw errcode(new Error('peerInfo must be an instance of peer-info'), ERR_INVALID_PARAMETERS)
-    }
+    // TODO: This is not a `peer-info` instance anymore, but an object with the data.
```

Comment:
Perhaps to avoid confusion, when we get rid of PeerInfo we should start calling variables `peerData`?

Reply:
Yes, I was having a hard time with this naming confusion. Good suggestion! 👍